### PR TITLE
another test

### PR DESCRIPTION
**Pull request by smacker**

### CODE REVIEW

File: hello.go [hunk 1]
```diff
@@ -5,3 +5,7 @@ import "fmt"
 func main() {
```
Comment (smacker) [inline hello.go:5]: The file has increased in 4 lines.

File: hello.go [hunk 2]
```diff
@@ -5,3 +5,7 @@ import "fmt"
...
 	fmt.Println("hello world")
 }
+
+func Another_Test_MustComplainApi_Hm_why_it_doesnt_complain_i_dont_know_okay_lets_make_this_line_too_long() int {
+	return 2
```
Comment (carlosms) [inline hello.go:10]: a

Comment (carlosms) [inline hello.go:10]: b

Comment (carlosms) [inline hello.go:10]: a

Comment (carlosms) [inline hello.go:10]: b

**Pull request closed without merging**